neuralforecast/core.py

```diff
@@ -380,9 +384,33 @@ def _prepare_fit_distributed(
             min_size=df.groupBy(id_col).count().agg({"count": "min"}).first()[0],
         )
 
+    def _prepare_iterative_fit(
```

Comment:
Wondering about this name, why is it called `iterative` specifically?
If I understand it correctly, perhaps more descriptive names would be `_prepare_fit_for_local_files` and `LocalFilesTimeSeriesDataset`?

Reply:
this is a good shout!